torch/ao/nn/sparse/quantized/linear.py

```diff
@@ -157,26 +159,24 @@ def set_weight_bias(self, w: torch.Tensor, b: Optional[torch.Tensor],
         self._packed_params.set_weight_bias(w, b, row_block_size, col_block_size)
 
     @classmethod
-    def from_float(cls, mod):
+    def from_float(cls, mod, row_block_size=1, col_block_size=4):
```

Comment:
We should not need the row/block col size here. This block just applies the mask. The mask should enforce the required sparsity pattern

Reply:
I see, from the point of view of keeping the API clean, should we do the following:
1. When we create a sparseLinear module, the fakesparse module should have all the information needed to specify the mask.
2. At convert, we only rely on the fake-sparse module
```python
 def from_float(cls, mod):
  sparsity_pattern = mod.fake_sparse.sparsity_config.sparsity_pattern # This is the shape of the sparse blocks
```

Reply:
Is this being addressed in a later PR?